lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -0,0 +1,93 @@
+require 'gds_api/test_helpers/json_client_helper'
+require 'gds_api/test_helpers/content_item_helpers'
+require 'gds_api/test_helpers/intent_helpers'
+require 'json'
+
+module GdsApi
+  module TestHelpers
+    module PublishingApiV2
+      include ContentItemHelpers
+
+      PUBLISHING_API_V2_ENDPOINT = Plek.current.find('publishing-api') + '/v2'
+
+      def stub_publishing_api_v2_put_content(content_id, body)
+        stub_publishing_api_v2_put(content_id, body, '/content')
+      end
+
+      def stub_publishing_api_v2_put_links(content_id, body)
+        stub_publishing_api_v2_put(content_id, body, '/links')
+      end
+
+      def stub_default_publishing_api_v2_put()
+        stub_request(:put, %r{\A#{PUBLISHING_API_V2_ENDPOINT}/content})
+      end
+
+      def stub_default_publishing_api_v2_put_draft()
+        stub_request(:put, %r{\A#{PUBLISHING_API_V2_ENDPOINT}/draft-content})
+      end
+
+      def stub_default_publishing_api_v2_put_intent()
+        stub_request(:put, %r{\A#{PUBLISHING_API_V2_ENDPOINT}/publish-intent})
```

Comment:
I believe that publish-intent isn't part of v2. Some apps might need to use the unscoped/unversioned publish-intent and the v2 endpoints until publishing-api can provide the behaviour to make it redundant.